CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Newtonsoft.Json;
+
+namespace CalculatorLibrary
+{
+
```

Comment:
According to code conventions the unnecessary empty lines should be removed before and after brackets.